README.md

```diff
@@ -33,7 +33,7 @@ Support for running FireCloud outside of the Broad is planned.
 
 ### Development Stack Setup
 
-1. Make sure you've run the `firecloud-setup.sh` script in the `firecloud-develop` repo to render the config files and run script.
+1. Make sure you've run the `firecloud-setup.sh` script in the `local` branch of the `firecloud-develop` repo to render the config files and run script.
```

Comment:
The script will be in all branches of the repo once Jackie's PR is merged in.

Reply:
That's good to know. I'll just remove this change then?